images/macos/software-report/SoftwareReport.Android.psm1

```diff
@@ -192,4 +192,4 @@ function Get-IntelHaxmVersion {
         "Package Name" = "Intel HAXM"
         "Version" = $Matches.Version
     }
-}
\ No newline at end of file
+}
```

Comment:
nit: please revert this file if no changes

Reply:
yes, will revert